Assets/unZENity-VR/Scripts/Importer/PhoenixImporter.cs

```diff
@@ -29,11 +29,13 @@ private void Update()
             if (_loaded) return;
             _loaded = true;
 
+            var G1Dir = SingletonBehaviour<SettingsManager>.GetOrCreate().GameSettings.GothicIPath;
```

Comment:
For Android we could directly use [Application.persistentDataPath](https://docs.unity3d.com/ScriptReference/Application-persistentDataPath.html) to specify the where should the app search for the game files like so
```suggestion
            if(Application.platform == RuntimePlatform.Android) {
                var G1Dir=Application.persistentDataPath;
            }
            else var G1Dir = SingletonBehaviour<SettingsManager>.GetOrCreate().GameSettings.GothicIPath;
```